src/main/java/codesquad/springcafe/controller/ArticleController.java

```diff
@@ -0,0 +1,43 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.model.Article;
+import codesquad.springcafe.repository.ArticleRepository;
+import java.util.Optional;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class ArticleController {
+
+    private final ArticleRepository articleRepository;
+
+    public ArticleController(ArticleRepository articleRepository) {
+        this.articleRepository = articleRepository;
+    }
+
+    @GetMapping("/qna/form")
+    public String qnaForm() {
+        return "/qna/form";
+    }
+
+    @PostMapping("/qna/form")
```

Comment:
```suggestion
    @PostMapping("/questions")
```
글 쓰기 URL은 이정도가 좋아보입니다.